src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/EntityViewerSidebarModal.scss

```diff
@@ -0,0 +1,28 @@
+@import 'src/styles/common';
+
+.entityViewerSidebarModal {
+  background: $white;
+  font-weight: $light;
+  position: relative;
+  overflow: auto;
+
+  h3 {
+    font-size: 14px;
+  }
+
+  p {
+    font-size: 13px;
+    margin-bottom: 30px;
+  }
+}
+
+.closeButton {
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  img {
+    height: 15px;
+    width: 15px;
+  }
```

Comment:
what img?

Reply:
This looks like old code. Will remove from TrackBarPanel too.